lib/dor/workflow/client/queues.rb

```diff
@@ -11,53 +11,70 @@ def initialize(requestor:)
 
         # Returns all the distinct laneIds for a given workflow step
         #
-        # @param [String] repo The repository the object resides in.  The service recoginzes "dor" and "sdr" at the moment
+        # @param [String] repo -- deprecated, ignored by workflow service
         # @param [String] workflow name
         # @param [String] process name
         # @return [Array<String>] all of the distinct laneIds.  Array will be empty if no lane ids were found
-        def lane_ids(repo, workflow, process)
-          uri = "workflow_queue/lane_ids?step=#{repo}:#{workflow}:#{process}"
+        def lane_ids(*args)
+          if args.count == 3
+            Deprecation.warn(
+              self,
+              '`#lane_ids` only takes two args: workflow name, & process/step name. This will raise an exception in Dor::Workflow::Client 4.0.0'
+            )
+          end
+          uri = "workflow_queue/lane_ids?step=#{args.first}:#{args.second}"
```

Comment:
If they are passing 3 args here, then this will be wrong. e.g. `step=dor...`

Reply:
Addressed this by calling `args.shift`.